mi-api/src/main.rs

```diff
@@ -26,7 +26,7 @@ use utoipa_swagger_ui::SwaggerUi;
 
 fn influence_route() -> Router<SharedState> {
     Router::new()
-        .route("/get", post(get_influences))
+        .route("/get", get(get_influences))
```

Comment:
```suggestion
        .route("/get/:query_user_id", get(get_influences))
```

missing path parameter